behavior_tree_msgs/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(behavior_tree_msgs)
+
+find_package(catkin REQUIRED COMPONENTS
+  std_msgs
```

Comment:
I don't see anywhere std_msgs is needed. I think it can be safely removed.

Reply:
The original authors use it in `src/dot_bt.cpp`

Reply:
`behavior_tree_msgs` is a different package though. It's not used in this package.